<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1336</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot2" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>521</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>10</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Liniar Congruentual Method with M &lt; 103</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Liniar Congruentual Method with big M</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot3" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot4" native="true">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>531</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Liniar Congruentual Method for MacLaren–Marsaglia method</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string> MacLaren–Marsaglia method itself</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1336</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
